settings.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+	// Gets URL of homepage
+	function homePageURL() {
```

Comment:
Moved function to within a class.

---
In reply to: [71235559](https://github.com/AzureAD/active-directory-b2c-wordpress-plugin-openidconnect/pull/3#discussion_r71235559) [](ancestors = 71235559)